vcfa/resource_vcfa_org.go

```diff
@@ -0,0 +1,243 @@
+			},
+			"display_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: fmt.Sprintf("Appears in the Cloud application as a human-readable name of the %s", labelVcfaOrg),
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Optional description",
+			},
+			"is_enabled": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Default:     true,
+				Description: fmt.Sprintf("Defines if the %s enabled. Defaults to 'true'", labelVcfaOrg),
+			},
+			"is_classic_tenant": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true, // Cannot be changed once created
+				Description: fmt.Sprintf("Defines whether the %s is a classic VRA-style tenant", labelVcfaOrg),
+			},
+			"managed_by_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: fmt.Sprintf("%s owner ID", labelVcfaOrg),
+			},
+			"managed_by_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: fmt.Sprintf("%s owner Name", labelVcfaOrg),
+			},
+			"org_vdc_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of VDCs belonging to the %s", labelVcfaOrg),
+			},
+			"catalog_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of catalog belonging to the %s", labelVcfaOrg),
+			},
+			"vapp_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of vApps belonging to the %s", labelVcfaOrg),
+			},
+			"running_vm_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of running VMs in the %s", labelVcfaOrg),
+			},
+			"user_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of users in the %s", labelVcfaOrg),
+			},
+			"disk_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of disks in the %s", labelVcfaOrg),
+			},
+			"can_publish": {
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: fmt.Sprintf("Defines whether the %s can publish catalogs externally", labelVcfaOrg),
+			},
+			"directly_managed_org_count": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: fmt.Sprintf("Number of directly managed %ss", labelVcfaOrg),
+			},
+		},
+	}
+}
+
+func resourceVcfaOrgCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	vcdClient := meta.(*VCDClient)
+	c := crudConfig[*govcd.TmOrg, types.TmOrg]{
+		entityLabel:      labelVcfaOrg,
+		getTypeFunc:      getOrgType,
+		stateStoreFunc:   setOrgData,
+		createFunc:       vcdClient.CreateTmOrg,
+		resourceReadFunc: resourceVcfaOrgRead,
+	}
+	return createResource(ctx, d, meta, c)
```

Comment:
Just double checking. When I create an org and apply finishes, I still see the progress bar in UI working for a minute or so. Is it just a UI representation delay? We are waiting for org related tasks to finish completely before returning?

Reply:
Looks like UI issue.